R/model.R

```diff
@@ -233,6 +235,7 @@ model_exp <- function(
   sigma_b3,
   shape,
   rate,
+  bound = 10,
```

Comment:
How universally applicable will this bound be?  We don't specify any other hyperparameters.  For consistency, should we omit a default argument?